### DISCUSSION TITLE
Welcome to BMW-K1200LT-ABS-Conversion Discussions!

### DISCUSSION THREAD

**Discussion by OstapFerensovych** [OstapFerensovych/BMW-K1200LT-ABS-Conversion#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DavidWilh** [comment]
Hello!

Its been a hot minute, but I am looking at diving into this project on a K1200S.
In theory it should be the same to do it on the S.

I know its a long shot you will even see or answer this, but the hope is there 😎 
The one thing im wondering is the connection between the new ABS module and the CAN converter you have designed and developed.
Do you have a wiring diagram for it or is it a matter of matching up the wiring between the different diagrams?

**Reply by OstapFerensovych** [reply]
Hi, you should match your S wiring to this connector
<img width="984" height="1180" alt="Screenshot_20260505_082851_Chrome" src="https://github.com/user-attachments/assets/cf5d1a1d-b6de-4242-bab1-2987d5c745f2" />

**Reply by DavidWilh** [reply]
Thank you for responding!

I will be documenting the process as I go along, so that I can contribute a little to the project as well.
Wiring from the bike to the CAN module is clear. What I am uncertain of is the connection to the new ABS module.
Did you just match up between the wiring diagrams? 

From my understanding of the documentation, the CAN module provides data to the Instrument Cluster etc. with speed signal and the alerts + rear and brake light.
For the Gen 2. ABS to work it will need power, ground, signal from ABS sensors and brake switches?

**Comment by OstapFerensovych** [comment]
Here is a basic wiring diagram.
<img width="669" height="520" alt="image" src="https://github.com/user-attachments/assets/01821381-64fc-4f9f-a16b-646cff9e89b7" />
iABS gen 2 has only sensor inputs, power input, and can bus

**Reply by DavidWilh** [reply]
You are a legend!

Time to source a new module and get the PCB fabricated.
Thank you!
